lib/admin_page/user/user_item.dart

```diff
@@ -0,0 +1,94 @@
+// ignore_for_file: must_be_immutable
+
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+
+import 'package:web/admin_page/user/user_detail_page.dart';
+import 'package:web/extention_function.dart';
+import 'package:web/model/user_data.dart';
+import 'package:web/route/router_url_name.dart';
+import 'package:web/values/app_strings.dart';
+
+class UserItem extends StatelessWidget {
+  UserItem({Key? key, this.user}) : super(key: key);
+  UserData? user;
+  double height = 0;
+  double width = 0;
+
+  @override
+  Widget build(BuildContext context) {
+    height = MediaQuery.of(context).size.height;
+    width = MediaQuery.of(context).size.width;
+    return Padding(
+      padding: EdgeInsets.only(left: height * 0.016, right: height * 0.016),
+      child: Card(
+        elevation: 5,
+        shape: RoundedRectangleBorder(
+          borderRadius: BorderRadius.circular(15.0),
+        ),
+        child: InkWell(
+          onTap: () {
+            Get.toNamed(RouterUrlName.layoutPage,arguments: user);
```

Comment:
again this should be part of your controller or presenter.  The navigation or data logic should not be present inside the UI/Page